### DISCUSSION TITLE
No physical student ID

### DISCUSSION THREAD

**Discussion by TeemuGus** [community/community#57565] [GitHub Education]
### Hi everyone,

I'm trying to apply for student campus. But on the part where it requires me to "take a picture" of my student id card there comes a problem.
I only have a digital card for the student ID and the "check up" won't accept a photo taken from phone screen where the ID is shown.

Do i have any other way to prove that i'm a student?

**Comment by mwt** [comment]
Transcripts work. You can use many different things that confirm you are a student.

**Reply by TeemuGus** [reply]
How?
On the "sign up" it only lets me take a picture with my webcam or if i'm using phone, then phones camera

**Reply by TithiB-del** [reply]
yea you have to use that to get a legible photo of the document.

**Reply by mwt** [reply]
Yeah, we don't have IDs either. I tell people to use their phone to take a picture of their laptop screen.

You can actually upload a jpeg too, but it's very picky about being a jpeg with a specific resolution and file size.

**Comment by samus-aran** [comment]
Hi @TeemuGus  👋

We're happy you're here!

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
